Doc(eos_designs): Add connected_endpoints fabric documentation (#2458)
aristanetworks/ansible-avd: ansible_collections/arista/avd/roles/eos_designs/doc/fabric-variables.md

enabling with feature would remove the unmatched trunk groups like this:

![Figure: Enable only_local_vlan_trunk_groups](../../../media/only_local_vlan_trunk_groups.png)

## Documentation variables

Some variables are available to control documentation generation. This can be useful

> is for instance the project has thousands of endpoints, to be able to disable

Reviewer: ```suggestion
when for instance, the project has thousands of endpoints and can disable
```
